pkg/services/provisioning/provisioning.go

```diff
@@ -254,6 +254,8 @@ func (ps *ProvisioningServiceImpl) ProvisionDatasources(ctx context.Context) err
 }
 
 func (ps *ProvisioningServiceImpl) ProvisionPlugins(ctx context.Context) error {
+	ps.mutex.Lock()
+	defer ps.mutex.Unlock()
```

Comment:
I am not sure why this is needed and why is it different from datasources or dashboards?

Reply:
thanks for the explanation, I'd add it to datasources as well just for the sake of similarity